### PR TITLE
Pcm 973

### PR DESCRIPTION
**Pull request by sshumake**
@renujhamtani @pbathia

### CODE REVIEW

File: app/chromed.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!--[if lte IE 8]>
+    <meta http-equiv="x-ua-compatible" content="IE=Edge"/>
+    <![endif]-->
+    <meta http-equiv="Content-Type" content="text/html; charset=US-ASCII" />
     <link href="/chrome_themes/nimbus/css/chrometwo.css" rel="stylesheet" />
     <script type="text/javascript" src="/webassets/avalon/j/lib/require.js"></script>
     <title></title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width">
```
Comment (pbathia) [inline app/chromed.html]: I think this line is for responsiveness of site, do we need to remove this for cache stuff?

**Comment by renujhamtani** [comment]
Looks good to me!

File: app/chromed.html (hunk 1 above)
Comment (sshumake) [inline app/chromed.html]: This is used for mobile development.  I have refactored this file to match our prod file which does not have this line

Comment (pbathia) [inline app/chromed.html]: Ok.

**Pull request merged**